swaglyrics/__main__.py

```diff
@@ -7,66 +7,104 @@
 from cli import lyrics, clear
 import spotify
 from tab import app
+import chrome
 
 
 def main():
-	# 	print(r"""
-	#  ____                     _               _
-	# / ___|_      ____ _  __ _| |   _   _ _ __(_) ___ ___
-	# \___ \ \ /\ / / _` |/ _` | |  | | | | '__| |/ __/ __|
-	#  ___) \ V  V / (_| | (_| | |__| |_| | |  | | (__\__ \
-	# |____/ \_/\_/ \__,_|\__, |_____\__, |_|  |_|\___|___/
-	#                     |___/      |___/
-	# 	""")
-	print('Updating unsupported.txt from server.')
-	with open('unsupported.txt', 'w', encoding='utf-8') as f:
-		response = requests.get('http://aadibajpai.pythonanywhere.com/master_unsupported')
-		f.write(response.text)
-	print("Updated unsupported.txt successfully.")
+    # 	print(r"""
```

Comment:
Not sure why indentation or quotes changed?

Reply:
I actually ran the file through python code formatter
so could be  due to it

Reply:
Please revert those :)

Reply:
can u mention where specific?

Reply:
Whichever you changed. Just make sure the code you add is pep8 compliant 👍

Reply:
Yupp.
I have used Black Code formatter. and it is pep8 compliant 🙂

Reply:
I meant the code you add should be compliant. Don't change the indents of already written code 👍